langdetect/detector.py

```diff
@@ -180,13 +180,17 @@ def _init_probability(self):
 
     def _extract_ngrams(self):
         '''Extract n-grams from target text.'''
+        RANGE = range(1, NGram.N_GRAM + 1)
```

Comment:
Would it be even faster, if we forced list here, instead of range object? N_GRAM is a small number, so it wouldn't increase memory cost.